src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -22,6 +23,23 @@ const buildIncludeUrl = (href, type) => {
   return `${process.env.SIMORGH_INCLUDES_BASE_URL}${withTrailingHref}${resolvers[type]}`;
 };
 
+const ampSupport = url => {
+  // An amp-image query parameter on the include path indicates an AMP version of the include is available
+  const hasAmpImageQueryString = new Url(url, true).query['amp-image'];
+  return !!hasAmpImageQueryString;
```

Comment:
Just for reference:

include that does not support amp: `/news/special/2016/newsspec_14813/content/iframe/gahuza/us-gop.inc?responsive=true&app-clickable=true&app-image=http://a.files.bbci.co.uk/worldservice/live/assets/images/2016/11/09/161109092836_us_election_2nddaymaps_winner_ws_62_v3.png` (no `amp-image` param)

Include that does support amp: /include/newsspec/21841-green-diet/gahuza/app?responsive=true&newsapps=true&app-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-asdf/app_launcher_ws_640_7ania.png&app-clickable=true&amp-clickable=true&amp-image-height=360&amp-image-width=640&amp-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-asdf/app_launcher_ws_640_7ania.png (has `amp-image` param)